tsconfig.json

```diff
@@ -1,12 +1,13 @@
 {
   "compilerOptions": {
+    "importHelpers": true,
     "lib": [
       "dom",
       "es5",
       "es2015",
       "es2015.promise"
     ],
-    "module": "commonjs",
+    "module": "ESNext",
```

Comment:
esnext module output enables webpack module concatenation which remove a lot of webpack boilerplate code